test/api.test.js

```diff
@@ -63,20 +64,26 @@ describe('API Tests', () => {
 
       [div, api] = await createElementAndApi(adjustViewSpacingConf, options);
 
-      const p = new Promise((resolve) => {
+      await new Promise((resolve) => {
         waitForElements(
           div,
           ['.tiled-plot-div', '.track-renderer-div', '.top-track-container'],
           resolve,
         );
       });
 
-      await p;
-
       const tiledPlotEl = div.querySelector('.tiled-plot-div');
       const trackRendererEl = div.querySelector('.track-renderer-div');
       const topTrackEl = div.querySelector('.top-track-container');
 
+      // Wait until these elements have stopped changing their size.
+      // Check every 20ms for up to 2 seconds
+      await waitForSizeStabilization(
+        [tiledPlotEl.parentNode, trackRendererEl, topTrackEl],
+        20,
+        2000,
```

Comment:
Maybe turn these into an [options object](https://www.codereadability.com/what-are-javascript-options-objects/) (e.g., `{ timeInterval: 20, maxTime: 2000 }`)? Could probably then remove the comment above.

Reply:
Javascript needs to get its act together and allow kwargs. But in the meantime, I took your suggestion and added an options object.

This function you commented on is gone but there is another, waitForScalesStabilized which follows a similar pattern when I updated.